### ISSUE TITLE
Custom page content/view loading recursively

### ISSUE THREAD

**Issue by etulossolute** [open-admin-org/open-admin#36]
**Describe the bug**
When loading a custom view by following this section in the docs:
https://open-admin.org/docs/en/content-layout
the whole admin layout loads again within the custom page, instead of just loading the view.

![recursive_load_problem](https://user-images.githubusercontent.com/114397096/200516122-bd1248de-0346-4bea-a14e-7285f7baa8b1.PNG)

This only happens if the view is loaded by clicking the menu item from the sidebar. If the content is accessed directly by its URL in the browser, the view loads correctly:

- Item1 > SubItem1 --> Incorrect: Recursive load of the menus within the page.
- Access http://localhost:8000/admin/map directly -> Correct, view loads fine.
(URLs in the browser end up being the same in both cases)

**To Reproduce**
1. Create a controller which loads a view.
```
<?php

namespace App\Admin\Controllers;

use App\Http\Controllers\Controller;
use OpenAdmin\Admin\Layout\Content;

class MapController extends Controller {

    public function index()
    {
        $content = new Content();
        $content->header('Test1');
        $content->description('Description...');
        $content->view('map');
        return $content;
    }
}
```

2. Add the route entry in "routes.php"
`$router->get('map', [\App\Admin\Controllers\MapController::class, 'index']);`

3. Add a menu entry for the route, using "map" as URI.
4. Flush route and application cache and load the custom page via the sidebar menu entry.

**Expected behavior**
Only the custom page or assigned view should load within the already existing menu and header, instead of loading everything again.
![recursive_load_problem_url_load](https://user-images.githubusercontent.com/114397096/200516389-fc0b7c4b-0880-4808-be58-60a0bb5f520a.PNG)

**System**
 - Open-admin version 1.0.22
 - PHP version 8.1.10
 - Laravel Version 9.34.0
 - OS: Windows 10
 - Browser: Google Chrome 107.0.5304.88

**Comment by open-admin-org** [comment]
Hi Etulossolute, 
Thanks for reaching out and the detail error report, and give open-admin a try. Lets try to solve this. I`m trying to reproduce this issue but i can't seem to find any issue with your code. Few thing we could try:

1. Did you to upgrade open-admin from an earlier version? if so could you run: 
`php artisan vendor:publish --tag=open-admin-assets --force`
and see if the problem is solved (force refresh to by-pass cache as well)

2.  If that didn't solve the issue, could you check if there is any javascript errors in the console.
3.  and last, could you share the content of your map view maybe the issues lies somewhere in there.

**Comment by etulossolute** [comment]
Hello, thank you for the prompt response.

I have updated the open-admin dependency via composer, going from 1.0.22 to 1.0.24 and published the files, but the behaviour remains the same.

Checking the console, I don't see any errors, so I guess there are none directly responsible for this.

The view I am trying to show is a compiled view made with Angular Typescript.
The resources that it is trying to load are available in the public folder of the application (and it works correctly, as previously stated, when not accessing via the menu item).

```
<!DOCTYPE html><html lang="en">
<head>
    <meta charset="utf-8">
    <title>AngularLeafletMaps</title>
    <base href="/assets/angular-leaflet-maps/">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="icon" type="image/x-icon" href="favicon.ico">
    <link rel="stylesheet" href="styles.e1a1f13160c9a76e.css" media="print" onload="this.media='all'">
    <noscript><link rel="stylesheet" href="styles.e1a1f13160c9a76e.css"></noscript>
</head>

<body>
<app-root></app-root>

<script src="runtime.0609582bebb27284.js" type="module"></script>
<script src="polyfills.63ea96a4f701c279.js" type="module"></script>
<script src="main.284a410913f8002f.js" type="module"></script>

</body>
</html>
```

If I try with a view made with plain HTML (no Angular app), the page loads correctly from the menu button, but loads standalone (without showing any of the menus or header) if the URL is accessed directly.

**Comment by open-admin-org** [comment]
Hi @etulossolute 

After a little testing i found that the script tags with a src-attribute where not being executed. I just released a new version 'v1.0.25' that loads script with src-attribute when the view is loaded. You might want make a view changes to you template though. The <base is likely to break the whole system, and the head / body / html tag are already present, so these should be removed as well. When you click the link the template will be loaded into the "#app" div. 

After updating run the `php artisan vendor:publish --tag=open-admin-assets --force` again

Probably the content of your blade file should be something like this:

`
<div class="card">
    <div class="card-header">Header</div>
    <div class="card-body">Body
        <link rel="stylesheet" href="/hello-world/styles.ef46db3751d8e999.css">
        <app-root>
        </app-root>
        <script src="/hello-world/runtime.5747c2d0904db68f.js"></script>
        <script src="/hello-world/polyfills.3495ce0a0cb55688.js"></script>
        <script src="/hello-world/main.c23c41217487f15c.js"></script>
    </div>
</div>
`

And alternative approach, could also be using an iframe.

**Comment by etulossolute** [comment]
Hello @open-admin-org

I have just updated to 1.0.25 and tried what you commented on the post. I see the definition of my Blade view was quite wrong now seeing your implementation. I now understand how it works.

It is working great. I just had to change the path of some assets to load correctly (like images for custom markers in the map). Also, the behaviour is now the same with both loading from the URL and clicking on the menu item.

![image](https://user-images.githubusercontent.com/114397096/201047280-6e86deb1-81ee-458d-bd17-b4d21684c059.png)

Thank you very much for your support.
